django/pyrog/api/serializers/import_export.py

```diff
@@ -87,9 +89,21 @@ class MappingOwnerSerializer(serializers.ModelSerializer):
 
     class Meta:
         model = Owner
-        fields = ["id", "name", "schema", "columns"]
+        fields = ["id", "name", "columns"]
+        read_only_fields = ["schema"]
         extra_kwargs = {"id": {"read_only": False}}  # Put `id` in validated data
 
+    def to_internal_value(self, data):
```

Comment:
document how import/export works in Import_export.py. eg: we don't take into account the db schema when importing  a mapping